src/components/pill-selector-dropdown/__tests__/PillSelectorDropdown-test.js

```diff
@@ -137,12 +137,12 @@ describe('components/pill-selector-dropdown/PillSelectorDropdown', () => {
             );
             const instance = wrapper.instance();
             wrapper.setState({
-                inputValue: 'aaron@example.com, bademail,hello@gmail.com',
+                inputValue: 'aaron@box.com, bademail,hello@gmail.com',
```

Comment:
Remove any Box related data from OSS projects, just in case any of that data is real :)

Reply:
Sure thing. Will fix.